workers/unity/Packages/com.improbable.gdk.core/Utility/BlittableBool.cs

```diff
@@ -2,6 +2,12 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     A blittable bool implementation to use in Unity's ECS.
+    /// </summary>
+    /// <remarks>
+    ///     This is required because bool is not blittable by default.
```

Comment:
Don't need to say it's required. "Can be used in place of bool where a blittable type is needed"